Bottom: 8a0bc35ec2704699ed4b1466444ed762e1a37f78
Top:    4c3f33567c0e8ad7cad13ebec360b90ea499a7c4
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-07 14:37:02 +0800

autofs-5.0.7 - fix incorrect value reference in parse_line()

This chack should clearly be on the contents of key not the pointer value.


---

diff --git a/lib/defaults.c b/lib/defaults.c
index 1e89509..7c65387 100644
--- a/lib/defaults.c
+++ b/lib/defaults.c
@@ -167,7 +167,7 @@ static int parse_line(char *line, char **res, char **value)
 	while (*key && *key == ' ')
 		key++;
 
-	if (!key)
+	if (!*key)
 		return 0;
 
 	if (!(val = strchr(key, '=')))
